tools/webpack

```diff
@@ -21,15 +21,15 @@ STATIC_PATH = 'static/'
 def run():
     # type: () -> None
     """Builds for production, writing the output to disk"""
-    subprocess.check_call(['node', 'node_modules/.bin/webpack'] +
+    subprocess.check_call(['node', 'node_modules/.bin/webpack-cli'] +
                           ['--config', 'tools/webpack.config.ts', '-p'] +
                           ['--env', 'production'])
 
 def run_watch(host, port, minify):
     # type: (str, str, bool) -> None
     """watches and rebuilds on changes, serving files from memory via webpack-dev-server"""
     webpack_args = ['node', 'node_modules/.bin/webpack-dev-server']
-    webpack_args += ['--config', 'tools/webpack.config.ts', '--watch-poll', '--port', port, "--host", host]
+    webpack_args += ['--config', 'tools/webpack.config.ts', '--progress', '--port', port, "--host", host]
```

Comment:
It great that the `watch-poll` flag was removed since its handled on the webpack config side but do we really want the progress flag? All it does right now is just `10% building modules 5/5 modules 0 active` and `78 modules`; just adds noise when running dev server.

Reply:
Yeah, agreed, it felt noisy to me in the `run-dev.py` output too.

Reply:
Sure I'll remove the flag.